pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -29,6 +29,28 @@ type rulesConnection struct {
 	egressRules  rulesInLayers
 }
 
+// LoadBalancerRule is a rule applied to all private IPs of a given load balancer:
+// these private IPs can only connect to pool members of the load balancer
+type LoadBalancerRule struct {
+	// the relevant load balancer:
+	lb LoadBalancer
+	//	Deny- true if src is pip, and dst is not pool member:
+	deny bool
+}
+
+func NewLoadBalancerRule(lb LoadBalancer, deny bool) *LoadBalancerRule {
+	return &LoadBalancerRule{lb, deny}
+}
+func (lbr *LoadBalancerRule) Deny() bool { return lbr.deny }
+
+func (lbr *LoadBalancerRule) String() string {
+	action := "allow"
+	if lbr.Deny() {
+		action = "blocks"
+	}
+	return fmt.Sprintf("load balancer %s %s connection to destinations which are its pool members\n", lbr.lb.Name(), action)
```

Comment:
Instead of
`load balancer kube-clusterid:1-3a95ce10fa514d48ab490f9eda1a6db1[LoadBalancer] allow connection to destinations which are its pool members`
Please write
`load balancer kube-clusterid:1-3a95ce10fa514d48ab490f9eda1a6db1[LoadBalancer] allows connection to <xyz> since it is one of its pool members`
Similarly:
In 
`Load Balancer: load balancer kube-clusterid:1-3a95ce10fa514d48ab490f9eda1a6db1[LoadBalancer] blocks connection to destinations which are its pool members `
1. No need for double Load balancer in the prefix
2. ".... blocks connection to <css> since its not one of its pool members"
Similarly in the detail of blocks and in the header of allows

Reply:
fixed